### ISSUE TITLE
Provide a way for the windows signing service to control name and url parameters of the attached signature

### ISSUE THREAD

**Issue by netomi** [eclipse-cbi/org.eclipse.cbi#503]
Currently, the name and url used for signing an executable is hard-coded in the service.

That means all executables signed with the service will have the same information which currently is set to the following:

```
      windows.jsign.description=Eclipse Foundation, Inc.
      windows.jsign.url=http://www.eclipse.org/

```

Now I am not aware of any reasons to do so, but I find this kind of odd. Shouldnt we support setting this information in the request as well?

@mbarbero  ?

**Comment by mbarbero** [comment]
You mean that this is not customizable by the caller of the service, correct? If this is only metadata and does not need to match the CN of the certificate, then I'm not opposed to it.

**Comment by netomi** [comment]
indeed, atm the caller of the service has no control over that parameters and always the fixed values are taken for every executable that is signed. I dont know what is displayed as information in Windows as even osslsigncode does not show that info surprisingly, so a second opinion on that would be good. But I assume that this is information that is specific to the executable that is signed, so having fixed values for everything is a bit couter intuitive imho.

**Comment by mbarbero** [comment]
Agreed.

**Comment by merks** [comment]
Thanks for being proactive.
